projects/packages/my-jetpack/_inc/components/product-cards-section/index.jsx

```diff
@@ -21,21 +22,20 @@ const { showJetpackStatsCard = false } = window.myJetpackInitialState?.myJetpack
  * @returns {object} ProductCardsSection React component.
  */
 const ProductCardsSection = () => {
-	const items = [
-		BackupCard,
-		ScanAndProtectCard,
-		AntiSpamCard,
-		BoostCard,
-		SearchCard,
-		VideopressCard,
-		CrmCard,
-		SocialCard,
-		AiCard,
-	];
+	const { productData, fetchingProductData } = useProductData();
 
-	if ( showJetpackStatsCard ) {
-		items.splice( 6, 0, StatsCard );
-	}
+	const items = {
+		backups: BackupCard,
+		scan: ScanAndProtectCard,
+		antispam: AntiSpamCard,
+		boost: BoostCard,
+		search: SearchCard,
+		videopress: VideopressCard,
+		stats: showJetpackStatsCard ? StatsCard : null,
```

Comment:
I am seeing the stats card at the top and a stats card in the product grid. I don't think that's caused by this PR, but seems like bug that we should clean up in another PR.

![Screenshot 2023-10-19 at 4 19 46 PM](https://github.com/Automattic/jetpack/assets/18016357/e22bbd34-e7b0-4646-ade6-32b78826690e)

Reply:
Added to the maintenance board